commit 566a99e128f58c48f0b431735d154c236b30dac1
Author: bertin <macportsuser@macports.org>
Date:   Sun May 25 22:21:59 2025 +0200

    /opt/local/site-ports/sysutils/MacPorts/files/patch-no-update-nagging.diff

diff --git a/src/macports1.0/macports.tcl b/src/macports1.0/macports.tcl
index 1c9fcb0b6..e65a09cb2 100644
--- a/src/macports1.0/macports.tcl
+++ b/src/macports1.0/macports.tcl
@@ -1933,9 +1933,9 @@ match macports.conf.default."
         set default_source_url [lindex $sources_default 0]
         if {[macports::getprotocol $default_source_url] in {file rsync}} {
             set default_portindex [macports::getindex $default_source_url]
-            if {[file exists $default_portindex] && [clock seconds] - [file mtime $default_portindex] > 1209600} {
-                ui_warn "port definitions are more than two weeks old, consider updating them by running 'port selfupdate'."
-            }
+#             if {[file exists $default_portindex] && [clock seconds] - [file mtime $default_portindex] > 1209600} {
+#                 ui_warn "port definitions are more than two weeks old, consider updating them by running 'port selfupdate'."
+#             }
         }
     }
 
